tools/build/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Dockerfile
+FROM ubuntu:xenial
```

Comment:
Target OS os Ubuntu xenial. It is not an issue for container, but in order to minimize potential issues with package